File: Python/Notes/conditionals.py
# JF Conditionals Notes

"""num = 1000

if num < 10 and num > -10:
    print(f"{num} is a single digit number")
elif num < 100:
    print(f"{num} is a double digit number")
else:
    print(f"{num} is a triple digit number")
    print(". . . .or is it?")
    if num**2 > 1000:
        print(f"Surprise number! {num**2}")"""

name = input("What is your name?: ").strip().title()

if name == "Arthur":
    print("Hello, King Arthur!")
    quest = input("What is your quest?: ").strip().capitalize()
    if "Holy Grail" in quest.title():
        print("Good luck!")
    else:
        print(f"{quest} is a lame quest.")
else:
    print(f"Hello {name}. You are not a king.")

File: C/Notes/conditionals.c
// JF Conditionals Notes

#include <stdio.h>
#include <string.h>
#include <stdbool.h>

int main(){
    bool is_programming = true;
    
    int grade = 75;
    if (grade >= 90) {
        printf("Your grade is %d percent, that is an A!\n", grade);
    } else if (grade >= 80) {
        printf("Your grade is %d percent, that is a B.\n", grade);
    } else if (grade >= 70) {
        printf("Your grade is %d percent, that is a C.\n", grade);
    } else if (grade >= 60) {
        printf("Your grade is %d percent, that is a D.\n", grade);
    } else {
        printf("Your grade is %d percent, that means you are failing.\n", grade);
    }

    int num = -442;
    if (num < 10 && num > -10) {
        printf("Your number is %d, which is a single digit number.\n", num);
    } else if (num < 100 && num > -100) {
        printf("Your number is %d, which is a double digit number.\n", num);
    } else if (num < 1000 && num > -1000) {
        printf("Your number is %d, which is a triple digit number.\n", num);
    } else {
        printf("Your number is %d, that's a pretty big number!\n", num);
    }

    char name[] = "Andrew";
    if (strcmp(name, "Andrew") == 0) {
        printf("Welcome Admin!\n");
    } else {
        printf("Welcome %s!\n", name);
    }

    return 0;
}